src/sentry/api/endpoints/organization_events_stats.py

```diff
@@ -34,6 +35,16 @@ def get(self, request, organization):
 
         snuba_args = self.get_field(request, snuba_args)
 
+        cache_key = cache_key_for_snuba_args(organization, params, snuba_args)
+
+        snuba_data = default_cache.get(cache_key)
+
+        import logging
+
+        if snuba_data is not None:
+            logging.info("foo cache: %s", cache_key)
```

Comment:
Should remove this. It would be good to have a datadog metric/or APM span to let us track how effective caching is.